R/repo-helper.R

```diff
@@ -91,3 +91,37 @@ set_usethis_desc <- function(maintainer, email, date) {
     .local_envir = parent.frame(n = 2)
   )
 }
+
+
+#' Run code in temporary project
+#'
+#' @inheritParams create_demo_project
+#' @param code Code to run with temporary active project
+#'
+#' @details If you open the project you are responsible for deleting it.
```

Comment:
How does this sound?
Otherwise once one gets to the project no file can be opened.

Reply:
or maybe we can remove the open argument.

Reply:
Let's remove `open`, we have `create_demo_project()` that exposes it. (If it's not exported, we could.)